pwa/src/components/objectEntities/ObjectEntitiesTable.tsx

```diff
@@ -161,7 +161,16 @@ const ObjectEntitiesTable: React.FC<ObjectEntitiesTableProps> = ({ entityId }) =
             <Modal
               title={`Create a new ${entity?.name} object`}
               id="objectModal"
-              body={() => <>{FormIO && FormIO}</>}
+              body={() => (
+                <>
+                  { FormIO && formIOSchema.components && FormIO }
+                  {
+                    entity?.handlers?.length < 1 && (
```

Comment:
This is basically a [falsy](https://developer.mozilla.org/en-US/docs/Glossary/Falsy?retiredLocale=nl) check. Right now, you're checking on `< 0`, in reality it would never be `-1`, it's minimum would be `0`.

Therefore, the check would more logically be `entity?.handlers?.length === 0 &&`. But because `0` is falsy, we can treat it as such (something like how we would treat a boolean), like so:

```
!entity?.handlers?.length && <>...</>
```